### PR TITLE
Documentation - Improved description for addLoading and removeLoading

### PR DESCRIPTION
**Pull request by Softwarecules**
## Summary

Enhancing the descriptions for `addLoading` and `removeLoading` and making sure they're the same at every point they're used.

## Instructions for local reproduction and review

Read the changed README files.

### CODE REVIEW

File: packages/plugins/AddressSearch/README.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ In `categoryProperties` and `groupProperties`, id strings called `groupId` and `
 | - | - | - |
 | searchMethods | searchMethodsObject[] | Array of search method descriptions. Only searches configured here can be used. |
 | afterResultComponent | VueConstructor? | If given, this component will be rendered in the last line of every single search result. It will be forwarded its search result feature as prop `feature` of type `GeoJSON.Feature`, and the focus state of the result as prop `focus` of type `boolean`. |
-| addLoading | string? | Optional loading action name to start loading. |
+| addLoading | string? | Expects the path to an action within the store. This action is committed with a loading key as the payload when starting reverse geocoding. |
```
Comment (warm-coolguy) [inline packages/plugins/AddressSearch/README.md:29]: This is not the reverse geocoder. I guess this is from a copy paste error. Also, actions can not be commited; only mutations can. Actions would require dispatchment.

I'd suggest the following change for a copyable field description that can be used in all places (with the replacement of "starting" with "finishing", where it fits).

```suggestion
| addLoading | string? | Expects the path to a mutation within the store. This mutation is committed with a plugin-specific loading key as payload when starting asynchronous procedures that are intended to be communicated to the user. |
```

File: packages/plugins/AddressSearch/README.md [hunk 2]
```diff
@@ -26,14 +26,14 @@ In `categoryProperties` and `groupProperties`, id strings called `groupId` and `
...
 | customSelectResult | Record<string, customSelectFunction>? | An object that maps categoryIds to functions. These functions are then called as vuex store actions instead of the `selectResult` default implementation. This allows overriding selection behaviour with full store access. Use `''` as key for categoryless results. |
 | focusAfterSearch | boolean? | Whether the focus should switch to the first result after a successful search. Defaults to `false`. |
 | groupProperties | Record<string, groupProperties>? | An object defining properties for a group. The searchMethod's groupId is used as identifier. All services without groupId fall back to the key `"defaultGroup"`. |
 | minLength | number? | Minimal input length after which searches are started. Defaults to 0. |
-| removeLoading | string? | Optional loading action name to end loading. |
+| removeLoading | string? | Expects the path to an action within the store. This action is commited with a loading key as the payload when finishing reverse geocoding. |
```
Comment (warm-coolguy) [inline packages/plugins/AddressSearch/README.md:36]:
```suggestion
| addLoading | string? | Expects the path to a mutation within the store. This mutation is committed with a plugin-specific loading key as payload when finishing asynchronous procedures that are intended to be communicated to the user. |
```

File: packages/plugins/ReverseGeocoder/README.md [hunk 3]
```diff
@@ -13,10 +13,10 @@ This module has been written for the HH WPS service. The return format is custom
 | fieldName | type | description |
 | - | - | - |
 | url | string | URL of the WPS to use for reverse geocoding. |
-| addLoading | string? | Points to an action in the store; commited with a loading key as payload on starting reverse geocoding. |
+| addLoading | string? | Expects the path to an action within the store. This action is committed with a loading key as the payload when starting reverse geocoding. |
```
Comment (warm-coolguy) [inline packages/plugins/ReverseGeocoder/README.md:16]:
```suggestion
| addLoading | string? | Expects the path to a mutation within the store. This mutation is committed with a plugin-specific loading key as payload when starting asynchronous procedures that are intended to be communicated to the user. |
```

File: packages/plugins/ReverseGeocoder/README.md [hunk 4]
```diff
@@ -13,10 +13,10 @@ This module has been written for the HH WPS service. The return format is custom
...
 | addressTarget | string? | Points to a path in the store where an address can be put. If given, ReverseGeocoder will update on resolve. If not given, ReverseGeocoder's results can only be retrieved by awaiting its action. |
 | coordinateSource | string? | Points to a path in the store where a coordinate may be. If given, ReverseGeocoder will resolve on any updates. If not given, ReverseGeocoder is passive and waits for its action to be called. Please mind that, when referencing another plugin, that plugin must be in `addPlugins` before this one. |
-| removeLoading | string? | Points to an action in the store; commited with a loading key as payload on finishing reverse geocoding. |
+| removeLoading | string? | Expects the path to an action within the store. This action is commited with a loading key as the payload when finishing reverse geocoding. |
```
Comment (warm-coolguy) [inline packages/plugins/ReverseGeocoder/README.md:19]:
```suggestion
| addLoading | string? | Expects the path to a mutation within the store. This mutation is committed with a plugin-specific loading key as payload when finishing asynchronous procedures that are intended to be communicated to the user. |
```

File: packages/plugins/LoadingIndicator/README.md [hunk 5]
```diff
@@ -28,9 +28,9 @@ map.$store.commit('plugin/loadingIndicator/removeLoadingKey', key)
 
 ![Loading indicator example](./readme_loadingIndicator_example.png)
 
-The key must be unique and is kept track of via a Set. It can't be added multiple times, and removing it once always removes it altogether. It is advised to use a key like `{my-plugin-or-application-name}-{procedure-name}` to avoid name conflicts. The LoadingIndicator will usually be used for asynchronous code.
+The key must be unique and is managed using a Set. This ensures that it can't be added multiple times, and removing it once will remove it completely. To avoid name conflicts, it is recommended to use a key format like `{my-plugin-or-application-name}-{procedure-name}`. The `LoadingIndicator` is typically used for asynchronous operations.
 
-As such, **always call `removeLoadingKey` in the `finally` section of your code** to prevent hanging loading indicators.
+Therefore, **always call `removeLoadingKey` in the `finally` block of your code** to prevent loading indicators from hanging.
```
Comment (warm-coolguy) [inline packages/plugins/LoadingIndicator/README.md:33]: I see no added value in this paraphrasing and propose the change to this README is dropped completely together with the CHANGELOG change. I also perceive this section as to be alright as it was previously.

Should any substantial changes be made, the CHANGELOG should refer to both mutations, as the documentation change pertains to both mutations as indicated by the example above.

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
🏓 @Softwarecules

**Comment by warm-coolguy** [comment]
https://github.com/Dataport/polar/pull/169/commits/c9a4d1e6fdfd79c6007a5d9851abba55d6b1ea38

Fixed a name mixup I brought in with suggestions.

File: packages/plugins/LoadingIndicator/README.md (hunk 5 above)
Comment (Softwarecules) [reply packages/plugins/LoadingIndicator/README.md]: @warm-coolguy 🏓

**Review by warm-coolguy — APPROVED** [review]
🚀 @Softwarecules

**Pull request merged**